src/Bridge/Symfony/XliffFileDumper.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp\Bridge\Symfony;
+
+use Symfony\Component\Translation\Dumper\FileDumper;
+use Symfony\Component\Translation\MessageCatalogue;
+
+/**
+ * Custom XliffFileDumper for use with phrase app
+ */
+class XliffFileDumper extends FileDumper
```

Comment:
The problem is not in the php source code - all is fine even with that crappy key. But when you look at the phraseapp website and want to do translations, the md5-hash is shown, and you have to translate md5 hashes! Nobody knows what is to do there, when you have md5-hashes given as input for translation. Our translation-team kills me, when I give then md5 hashes to translate. The `source` from the xliff file is not even shown on the phraseapp website.